### PR TITLE
Feature/profile auth

### PR DESCRIPTION
**Pull request by letrylesha**
image
image

### CODE REVIEW

**Review by danill1278 — COMMENTED** [review]
nice profile page

**Review by CBond05 — APPROVED** [review]
seems ok

File: src/components/App/Auth/Auth.tsx [hunk 1]
```diff
@@ -1,44 +1,87 @@
...
+  background-position: -100px 0px;
+  box-shadow: rgba(36,195,249, .4) 0 0 0 100vw inset;
+
+  @media (max-width: 1300px) {
+    width: 40%;
+  }
+  @media (max-width: 1000px) {
+    display: none;
   }
 `;
```
Comment (drews-believer) [inline src/components/App/Auth/Auth.tsx:29]: move to style file ?))

**Review by drews-believer — APPROVED** [review]

File: src/components/App/Auth/Auth.tsx (hunk 1 above)
Comment (letrylesha) [reply src/components/App/Auth/Auth.tsx:29]: Da, I will

**Review by yanosh-almashi — APPROVED** [review]

**Review by danill1278 — APPROVED** [review]

**Pull request merged**